docs/document/content/features/encrypt/use-norms.cn.md

```diff
@@ -11,5 +11,6 @@ weight = 2
 ## 不支持项
 
 * 需自行处理数据库中原始的存量数据；
+* 加密字段无法支持查询不区分大小写功能；
```

Comment:
@acmenlt I think this is a bug that needs to be fixed.

Reply:
Mysql database queries are case insensitive by default. Case sensitive after encryption. I don't think this is a problem. It's correct.

Reply:
> Mysql database queries are case insensitive by default. Case sensitive after encryption. I don't think this is a problem. It's correct.

I got it.